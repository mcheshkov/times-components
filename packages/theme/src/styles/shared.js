const sharedStyle = {
  ThemeBody: {
    padding: 20
  }
};

export default sharedStyle;
