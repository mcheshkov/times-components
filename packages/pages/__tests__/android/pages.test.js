import shared from "../shared";

describe("Pages tests on android", () => {
  shared();
});
