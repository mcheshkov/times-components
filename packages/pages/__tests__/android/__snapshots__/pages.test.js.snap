// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages tests on android renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexBasis": "100%",
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color={undefined}
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
